src/controllers/userController.ts

```diff
@@ -116,6 +117,8 @@ class UserController {
         });
         return;
       }
+      user.wallet += 10; // Reward user with 10 coins for each daily check-in
```

Comment:
This logic was supposed to be implemented on line 137 as we would have made the necessary checks to see whether the user is checking in for today, if he has already checked in and when was the last time the user checked in.
I put a placeholder logic that you can update with this logic
```suggestion
      if (user.lastLogin.toDateString() === yesterday.toDateString()) {
        // Consecutive login
        user.wallet += 10; // Reward user with 10 coins for each daily check-in
      } 
```